plugins/inlinetoolbar/plugin.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict';
+
+	/**
+	 * A class that represents a inline toolbar capable of presenting defined
+	 * menu buttons.
+	 *
+	 * @class
+	 * @extends CKEDITOR.ui.balloonPanel
+	 * @constructor Creates a command class instance.
+	 * @since 4.8
+	 * @param {CKEDITOR.editor} editor The editor instance for which the panel is created.
+	 * @param {Object} definition An object containing the panel definition.
+	 */
+	CKEDITOR.ui.inlineToolbarView = function( editor, definition ) {
+		CKEDITOR.ui.balloonPanel.call( this, editor, definition );
+		this.listeners = [];
+	};
+
+	CKEDITOR.inlineToolbar = function( editor ) {
+		this.inlineToolbar = new CKEDITOR.ui.inlineToolbarView( editor );
+	};
+
+	CKEDITOR.plugins.add( 'inlinetoolbar', {
+		requires: 'balloonpanel',
+		init: function() {
+			CKEDITOR.ui.inlineToolbarView.prototype = CKEDITOR.tools.extend( {}, CKEDITOR.ui.balloonPanel.prototype );
+			/**
+			 * build inline toolbar DOM representation.
```

Comment:
Sentence should start with a capitalized letter. Also one extra line of space between the comment and previous line should be kept.